full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts

```diff
@@ -574,18 +567,20 @@ describe("iModelHub iModelsHandler", () => {
     mockGetIModelByName(projectId, newName, newDescription, imodel.id);
     updatediModel = (await iModelClient.iModels.get(requestContext, projectId, new IModelQuery().byName(newName)))[0];
```

Comment:
This could also cause issues if code fails before it's renamed to oldName.

Reply:
Same fix as in the case above